app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -225,6 +227,13 @@ class PatientSummaryScreen(
         .map { PatientSummaryLinkIdCancelled }
   }
 
+  private fun phoneNumberClicks(): Observable<UiEvent> {
+    return if (config.callPatientFeatureEnabled)
```

Comment:
Is this a feature flag or developer flag?
If this flag will be removed once this feature is complete then its alright.
If this will go to production and can be toggled, then let's move this logic to `Update` class. Wdyt?

Reply:
This is technically a developer flag, but I added it as a remote flag since we can turn it on on the `SBX` environment and review the intermediate feature.

But the final PR of this feature will remove this flag and enable the feature permanently, so I decided to leave this in the screen level. Does this make sense?